ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/logic/switch-evpn-route-clients.j2

```diff
@@ -0,0 +1,75 @@
+{% if switch.evpn_role == "server" %}
+{%     set switch.evpn_route_clients = {} %}
+{# #}
+{# Look for switches pointing to us as evpn_route_server #}
+{%     for some_switch in groups[fabric_name] if some_switch not in switch.evpn_route_servers | arista.avd.natural_sort %}
```

Comment:
I would suggest renaming `some_switch` var to `fabric_switch`

Reply:
Done! Thanks.